src/api/searchApi.ts

```diff
@@ -517,9 +515,12 @@ export const fetchResults = async (params: FetchResultsParams, signal?: AbortSig
   if (params.query) {
     searchParams.set(ResultParam.Query, params.query);
   }
-  if (params.scientificIndex) {
-    searchParams.set(ResultParam.ScientificReportPeriodBeforeParam, (+params.scientificIndex + 1).toString());
-    searchParams.set(ResultParam.ScientificReportPeriodSinceParam, params.scientificIndex);
+  if (params.scientificReportPeriodSince) {
+    searchParams.set(
+      ResultParam.ScientificReportPeriodBeforeParam,
+      (+params.scientificReportPeriodSince + 1).toString()
```

Comment:
skal det fortsatt være +1 her? 🤔 Oppsto ikke problemet fordi vi gjorde endringer her som ikke var gjenspeilet i URLen som eksporten bruker?

Reply:
Det sjekkes ikke etter `scientificReportPeriodBefore`, det settes kun i URL'en for eskport. Så for at de skal stemme overens må verdien settes her for å samkjøre url'en og api-kallet

Reply:
Ah, ser at det legges til +1 i URLen også for `scientificReportPeriodSince`, så da blir det jo riktig ja.
Kan argumentere for at dette skulle vært satt hver for seg da, siden de egentlig er uanvhengige av hverandre:
```
  if (params.scientificReportPeriodSince) {
    searchParams.set(ResultParam.ScientificReportPeriodSinceParam, params.scientificReportPeriodSince);
  }
  if (params.scientificReportPeriodBefore) {
    searchParams.set(ResultParam.ScientificReportPeriodBeforeParam, params.scientificReportPeriodBefore);
  }
```
your call